doc/builders/special/darwin-builder.section.md

```diff
@@ -61,3 +61,37 @@ builders-use-substitutes = true
 ```ShellSession
 $ sudo launchctl kickstart -k system/org.nixos.nix-daemon
 ```
+
+## As a NixOS module
+
+The `darwin.builder` bootstrapper runs a NixOS module with sensible defaults for
```

Comment:
What is a bootstrapper? I think you refer to the first sentence of the section, but this sounds like you're referring to something else.

```suggestion
The `darwin.builder` package wraps a NixOS module with sensible defaults for
```

Also modules don't run. They get evaluated, but that's not even relevant here.